frontend/modules/sequences/sequences.sagas.ts

```diff
@@ -116,20 +114,20 @@ export function* setSelectedDate({date}) {
 }
 
 export function* initializeSequences() {
-	const ifcSpacesHidden = yield select(selectIfcSpacesHidden);
-	if (ifcSpacesHidden) {
-		yield put(TreeActions.hideIfcSpaces());
+	const hiddenGeometryVisible = yield select(selectHiddenGeometryVisible);
+	if (!hiddenGeometryVisible) {
+		yield put(TreeActions.showHiddenGeometry());
 	}
 
-	yield put(SequencesActions.setIfcSpacesHidden(ifcSpacesHidden));
+	yield put(SequencesActions.setHiddenGeometryVisible(!hiddenGeometryVisible));
```

Comment:
This line is throwing an error @mkubiakapptension

Reply:
Also this line is throwing an error:
https://github.com/3drepo/3drepo.io/blob/30b3fc306ea6aed6b62bfa0ed956c7186452e25a/frontend/modules/viewpoints/viewpoints.sagas.ts#L227

Reply:
Thanks for spotting this @sanmont3drepo. I've pushed the fixes.